tests/internal/e2e/test/wdio.conf.ts

```diff
@@ -309,207 +315,317 @@ export const config: Options.Testrunner = {
+          );
+          await promisifiedExec(
+            `copy /y ${path.join(
+              repoRoot,
+              "package.json"
+            )} ${extensionUnderTest}`
+          );
+        });
+      } else {
+        foldersToCopy.forEach(async (folderName) => {
+          const folderPath = path.join(repoRoot, folderName);
+          await promisifiedExec(`cp -r ${folderPath} ${extensionUnderTest}`);
+        });
+        await promisifiedExec(
+          `cp ${path.join(repoRoot, "package.json")} ${extensionUnderTest}`
+        );
+      }
+    }
+
+    /**
+     * Builds multiple envs for release 23 and release 24
+     */
+    async function setupMultipleEnvironments() {
+      // Old and new VectorCAST versions
+      const vcastRoot = path.join(process.env.HOME, "vcast");
+      const oldVersion = "release23";
+      const newVersion = "release24";
+
+      // Total amount of envs to be build
+      const totalEnvCount = 4;
+
+      const placeholder = "%%REPLACE%%";
+      const unit = "unit";
+
+      const workspacePath = path.join(__dirname, "vcastTutorial");
+      const envTemplate = path.join(workspacePath, "TEST.template");
+      const unitFile = path.join(workspacePath, "unit.cpp");
+
+      const envTemplateContent = `
+ENVIRO.NEW
+ENVIRO.NAME: ${placeholder}
+ENVIRO.STUB_BY_FUNCTION: ${unit}
+ENVIRO.WHITE_BOX: NO
+ENVIRO.VCDB_FILENAME:
+ENVIRO.COVERAGE_TYPE: NONE
+ENVIRO.LIBRARY_STUBS:
+ENVIRO.STUB: ALL_BY_PROTOTYPE
+ENVIRO.COMPILER: CC
+ENVIRO.SEARCH_LIST: .
+ENVIRO.END
+`;
+
+      // Write template and cpp to file
+      await writeFile(envTemplate, envTemplateContent.trim());
+      const unitFileContent = "void foo(void) {}";
+      await writeFile(unitFile, unitFileContent);
+
+      // Create correct dir for the tests
+      await mkdir(workspacePath, { recursive: true });
+
+      // Create envs and copy them to workspacePath
+      for (let i = 1; i <= totalEnvCount; i++) {
+        // ENV_0i
+        const envName = `ENV_${i.toString().padStart(2, "0")}`;
+        const envFile = path.join(workspacePath, `${envName}.env`);
+
+        const envContent = (await readFile(envTemplate))
+          .toString()
+          .replace(placeholder, envName);
+        await writeFile(envFile, envContent);
+      }
+
+      // Initial build with the old version
+      process.env.VCAST_FORCE_OVERWRITE_ENV_DIR = "1";
+      process.env.VECTORCAST_DIR = path.join(vcastRoot, oldVersion);
+
+      const createCFG = `cd ${workspacePath} && ${process.env.VECTORCAST_DIR}/clicast -lc template GNU_CPP17_X`;
+      await promisifiedExec(createCFG);
+
+      // Build environments based on the iteration index
+      for (let i = 1; i <= totalEnvCount; i++) {
+        // ENV_0i
+        const envName = `ENV_${i.toString().padStart(2, "0")}`;
+        const envFile = path.join(workspacePath, `${envName}.env`);
+
+        // Switch VectorCAST version based on iteration (build 1,3 --> release 23, 2,4 --> release 24)
```

Comment:
maybe adding a VectorCAST version indicator to environment names could make this less confusing and remove the need for the comment, though it's clear what's going on

Reply:
Added release prefix to envs in 7c690c6fda33b48239ce5fb7aeb15024a3df53d6.